src/modules/tooltip/TooltipModule.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-    <TooltipBox />
+    <div>
+        <TooltipBox />
+    </div>
```

Comment:
I don't think that's necessary anymore (might be a relic of us wanting to add this location popup at this location)